wsd/wopi/CheckFileInfo.cpp

```diff
@@ -87,6 +87,8 @@ void CheckFileInfo::checkFileInfo(int redirectLimit)
         const std::string& wopiResponse = httpResponse->getBody();
         const bool failed = (httpResponse->statusLine().statusCode() != http::StatusCode::OK);
 
+        // here we get wopiResponse
```

Comment:
Well, actually, we got the response 3 lines above. "here" we process and parse it.